.gitignore

```diff
@@ -1,2 +1,7 @@
+# Website generation artifacts
 node_modules
-website-preview
\ No newline at end of file
+website-preview
+
+# Jetbrains IDEs
+.idea/
+*.iws
```

Comment:
🙏 I just need to stop worrying about me fat-fingering it and open a PR with a bunch of Jetbrains files in it.